docs/adr/0002-deployment-of-frontend.md

```diff
@@ -0,0 +1,23 @@
+# ADR 1: Deployment of static frontend
+
+Date: 2020-08-17
+
+## Status
+
+Pending
+
+## Context
+
+There are two purposed ways to setup the frontend and backend of the TTADP system.
+
+ 1. The backend hosts the static frontend directly; there is one app. The single app will require one Authority to Operate (ATO). Some tooling can easily be shared between the frontend and backend. Also synchronized deploys will be easier to accomplish with a single app.
+
+ 2. The backend and frontend are two separate apps, the backend a node express app and the frontend static HTML, javascript and CSS. Two apps allow for greater separation between the two components. There will need to be two ATOs. Tooling will be harder to share when the apps are split into two.
```

Comment:
We won't need two ATOs, it'll just be more to document. It's still a single "system" even split into separate cloud.gov apps.